test/webserv/integration/test_http_post.py

```
@@ -0,0 +1,137 @@
+import os
+
+import pytest
+from common_functions import read_file, send_request_and_assert_response
+from common_response import *
```

Ruff

`from common_response import *` used; unable to detect undefined names

(F403)